### PR TITLE
feat(tracking): add async tracking snippet

### LINKED ISSUE #29: Update Analytics Tracking
The current analytics approach loads the old GA snippet near the bottom of the page. This approach was originally created before browsers could use the `async` attribute to download and run scripts in parallel without blocking the parser. Because Hugo internal templates provide the async tracking snippet we should use it as it helps with dogfooding.

PR to follow.

### PR DESCRIPTION
**Pull request by ghost**
uses the internal async tracking snippet, recommended by google

resolves #29

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -2,6 +2,8 @@ baseurl = "http://themes.gohugo.io"
 languageCode = "en-us"
 title = "Hugo Themes"
 
+[params]
+googleAnalytics = "UA-7131036-4"
```
Comment (digitalcraftsman) [inline config.toml]: Please move `googleAnalytics` out of the `params` block. [`googleAnalytics`](https://github.com/spf13/hugo/blob/90d3fbf1da93a279cfe994a226ae82cf5441deab/tpl/tplimpl/template_embedded.go#L252) is site-wide variable like title.

Comment (ghost) [reply config.toml]: All set.

**Comment by digitalcraftsman** [comment]
Merged as 8d96be16e5cdf352ae17c98a62afc565ee648405.

**Comment by digitalcraftsman** [comment]
tn

**Comment by digitalcraftsman** [comment]
Merged as 55f279e.

**Pull request closed without merging**